services/QuillLMS/app/workers/clean_concept_result_instructions_worker.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class CleanConceptResultInstructionsWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: SidekiqQueue::MIGRATION
+
+  def perform(start, stop)
+    stop ||= ConceptResult.maximum(:id)
```

Comment:
`stop` should never be nil, so the conditional assignment is not necessary

Reply:
I've been using the ability to pass `nil` as the value for `stop` in testing, but you're probably right that it's not necessary to keep in for the final version.